src/Bot/Log.hs

```diff
@@ -49,6 +55,18 @@ recordUserMsg sender msg = do
         }
   return ()
 
+getRecentLogs :: Seconds -> Effect [LogRecord]
+getRecentLogs offset = do
+  currentTime <- now
+  let diff = intToNominalDiffTime $ -1 * offset
+  let startDate = addUTCTime diff currentTime
+  -- TODO there should be comparing selector when implemented
+  allLogs <- selectEntities "LogRecord" $ SortBy timestampPV Desc All
```

Comment:
Extracting all of the LogRecord-s seems to be dangerous. The current "production" database that I use on my streams contains almost 50000 of records. I suggest to select only last 100 records as a temporary limitation until we implement the comparing selector you mentioned in the TODO above.